components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -1153,6 +1153,8 @@ private ConfigParameters() {
     }
 
     public static final String API_LC_ACTION_DEPRECATE = "Deprecate";
+    public static final String DEPRECATE_CHECK_LIST_ITEM = "Deprecate old versions after publish the API";
```

Comment:
publishing*

Reply:
This text is taken from the life cyle xml config. We can not change this text. It should exactly match the exact value in  life cycle config. Other wise we have to migrate all the existing data.